test/integration/everything_ok_alarm_test.rb

```diff
@@ -1,9 +1,9 @@
-require 'test_helper'
+require 'application_system_test_case'
 
-class EverythingOkAlarmTest < ActionDispatch::IntegrationTest
+class EverythingOkAlarmTest < ApplicationSystemTestCase
   # confirm that the app is loading and that the main page is clean
   it 'SHOULD SOUND THIS ALARM WHEN EVERYTHING IS OKAY' do
-    get '/'
-    assert_response :redirect
+    visit root_path
+    assert_selector "h3", text: "Sign in"
```

Comment:
This is dead on -- we don't have `get` anymore except in controller tests and want to use the capybara goods now.